apps/designer/app/designer/features/topbar/menu/menu.tsx

```diff
@@ -140,9 +140,9 @@ export const Menu = ({ config, publish }: MenuProps) => {
         <DropdownMenuItem
           css={menuItemCss}
           onSelect={() => {
-            publish<"shortcut", string>({
+            publish({
               type: "shortcut",
-              payload: "undo",
+              payload: { name: "undo" },
```

Comment:
This was somewhat surprising - but as far as I could tell those call sites here probably were buggy as the subscriber expects an object:
https://github.com/webstudio-is/webstudio-designer/blob/b913ac73d726ad41ca33bda2d44f1bd6fdf09ea9/apps/designer/app/canvas/shared/use-shortcuts.ts#L167